cypress/e2e/settings/security-user-settings-tab.spec.ts

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Suguru Hirahara
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+
+describe("Security user settings tab", () => {
+    let homeserver: HomeserverInstance;
+
+    afterEach(() => {
+        cy.stopHomeserver(homeserver);
+    });
+
+    it("should be rendered properly", () => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, "Hanako");
+        });
+
+        cy.openUserSettings("Security");
+    });
+
+    describe("with posthog enabled", () => {
+        beforeEach(() => {
+            // Enable posthog
+            cy.intercept("/config.json?cachebuster=*", (req) => {
+                req.continue((res) => {
+                    res.send(200, {
+                        ...res.body,
+                        posthog: {
+                            project_api_key: "foo",
+                            api_host: "bar",
+                        },
+                        privacy_policy_url: "example.tld", // Set privacy policy URL to enable privacyPolicyLink
+                    });
+                });
+            });
+
+            cy.startHomeserver("default").then((data) => {
+                homeserver = data;
+                cy.initTestUser(homeserver, "Hanako");
+            });
+
+            // Hide "Notification" toast on Cypress Cloud
+            cy.contains(".mx_Toast_toast h2", "Notifications")
+                .should("exist")
+                .closest(".mx_Toast_toast")
+                .within(() => {
+                    cy.get(".mx_Toast_buttons .mx_AccessibleButton_kind_danger_outline").click(); // Click "Dismiss"
```

Comment:
Is it possible to click a button with the name „Dismiss“ here?
Then the comment would be unnecessary.

Reply:
You're right- I addressed it with https://github.com/matrix-org/matrix-react-sdk/pull/10925/commits/6df45a407c36de8631c7ba2a58d9a18673637944.